ceph.spec.in

```diff
@@ -153,7 +153,7 @@
 # do not provide gcc-annobin.so anymore, despite that they provide annobin.so. but
 # redhat-rpm-config still passes -fplugin=gcc-annobin to the compiler.
 %undefine _annotated_build
-%if 0%{?rhel} >= 8
+%if 0%{?rhel} == 8
```

Comment:
i believe that https://github.com/ceph/ceph/pull/47538/commits/75dd92353381c6ec5ac7251585a276059b8a2253 is a better fix though.

Reply:
Will it install gcc-toolset-12 on el9 though?

Reply:
please see the commit message of https://github.com/ceph/ceph/pull/47538/commits/edf07165a44eb73f3bbbae0c300d7f4f1bba1204 in  #47538

Reply:
gcc-toolset-11 is not available in CentOS 9 though.  Does that not matter?

Reply:
> gcc-toolset-11 is not available in CentOS 9 though. Does that not matter?

Guess not.  It built.